README.md

```diff
@@ -87,15 +87,20 @@ data "hivelocity_product" "tampa_product" {
 resource "hivelocity_bare_metal_device" "tampa_server" {
     product_id = "${data.hivelocity_product.tampa_product.product_id}"
     os_name = "CentOS 7.x"
-    location_name = "${data.hivelocity_product.tampa_product.location}"
+    location_name = "${data.hivelocity_product.tampa_product.data_center}"
```

Comment:
Do this change is required? I am asking because this is a breaking change, right?

Reply:
yes, the name location was changed to data_center on the endpoint. Was changed the product/list to use the FastInventoryService.flat_inventory and this returns "data_center".